modules/oidc-config-and-provider/main.tf

```diff
@@ -0,0 +1,135 @@
+resource "rhcs_rosa_oidc_config" "oidc_config" {
```

Comment:
I wonder if we should separate the module even further and have one module for managed and one for unmanaged, so that's clearer to the customer

Reply:
I don't think that having both in the same module might cause the customer to be more confused. As the cli also has one call with --managed flag